go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/spacemeshos/fixed v0.0.0-20210523192743-8d17e03c169a
 	github.com/spacemeshos/go-scale v1.1.7
 	github.com/spacemeshos/merkle-tree v0.2.1
-	github.com/spacemeshos/poet v0.7.0
+	github.com/spacemeshos/poet v0.7.1-0.20230405184528-298199ded0e6
```

Comment:
TODO(mafa): release new poet version and update

@poszu where do I update the used poet version in systests?

Reply:
systest/Makefile

Reply:
can you refer to the commits in this poet version update for me to understand? thanks